stock_release_channel/models/stock_release_channel.py

```diff
@@ -237,6 +244,7 @@ def _compute_picking_count(self):
 
             for record in self:
                 record[field] = count.get(record.id, 0)
```

Comment:
```suggestion
                record_group_data = count.get(record.id, {})
                record[field] = record_group_data and record_group_data["release_channel_id_count"]
```

Therefore line 226 or in your change line 233 must be changed, add the row instead of row["release... to the count dict